src/os/os_unix_test.go

```diff
@@ -0,0 +1,57 @@
+// +build darwin linux,!baremetal freebsd,!baremetal
+
+package os_test
+
+import (
+	"os"
+	. "os"
```

Comment:
Any reason to import os twice?

Reply:
Well.... the 2nd one is forced, but the first one is so I can use os. as a prefix on os functions from the test code.

Reply:
What do you mean, "forced"?
Either you can `import "os"` and use `os.Mkdir` etc or you can `import . "os"` and call `Mkdir`. Mixing the two seems confusing to me.

Reply:
Saw it in https://golang.org/src/os/os_test.go and thought it was normal.  Nevermind.

Reply:
Huh, weird. I don't know why they're doing that...